cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,79 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let user;
+
   before(() => {
+    cy.visit('https://demoqa.com/automation-practice-form');
 
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
+  it('should allow the registration of new student', () => {
```

Comment:
I think you should avoid so many spaces in this assignment

Reply:
done)